src/makeInfinityAdder.js

```diff
@@ -6,7 +6,25 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+  let sumIsShowed = false;
```

Comment:
you don't need `sumIsShowed` variable, checking `number` argument in the function is enough